README.md

```diff
@@ -1,8 +1,8 @@
 # Custom CSS and JS
 
-### **SPECIAL NOTE: If Visual Studio Code complains about that it is corrupted, simply click “Don't show again”.**
```

Comment:
Do not change these "titles" to list. Otherwise, people will not read them and forget these important instructions.

Reply:
I will restore them now.